features/proxy_config.feature

```diff
@@ -535,15 +533,15 @@ Feature: Proxy configuration
 
     @slow
     Scenario Outline: Attach command when authenticated proxy is configured manually for uaclient
-        Given a `<release>` `<machine_type>` machine with ubuntu-advantage-tools installed
-        Given a `focal` machine named `proxy`
+        Given a `<release>` `lxd-container` machine with ubuntu-advantage-tools installed
```

Comment:
I think this line should stay as it was
```suggestion
        Given a `<release>` `<machine_type>` machine with ubuntu-advantage-tools installed
```

Reply:
@orndorffgrant My rationale for this is if we want to expand this test in the future for VMs, the proxy machine should still be a `lxd-container`. But let me know if you disagree

Reply:
@lucasmoura I totally agree with the change to the `Given` line setting up the proxy, but the "machine with ubuntu-advantage-tools` should be <machine_type> still, right?

Reply:
Ohhh good catch, I missed that I changed that on both places. Fixing it now